### DISCUSSION TITLE
Issue with webpack / chunked file names

### DISCUSSION THREAD

**Discussion by notchris** [payloadcms/payload#2307] [Q&A]
# Bug Report

Just started seeing this issue after updating my page collection. I simply cleaned up my block imports, so not sure what could have caused this. I have a "Page" collection with two blocks. I moved the blocks to collections/blocks and updated my imports. Then I restarted our payload instance (on both servers as we have a load balancer). Not sure if editing the collection cause the issue, but now when I go to view the collection or edit an item, I get a bunch of 404's and cant access the content (blank page) every couple page loads.

![Screenshot 2023-03-13 112556](https://user-images.githubusercontent.com/1944433/224748032-0ff1dab0-0b23-4a7c-8383-795058bbcba7.png)

![Screenshot 2023-03-13 113028](https://user-images.githubusercontent.com/1944433/224749299-1cd8dd89-4a01-40c8-b4ef-144890d28bcc.png)

I'm thinking maybe this is an issue with webpack / page chunking / files being served from the load balancer. It's just odd because I never ran into it before so not sure what caused this.

## Steps to Reproduce

1. Hard to reproduce because of our setup with the load balancer.

**Comment by jmikrut** [comment]
Hey @notchris —hmm, we haven't changed anything with Webpack in a minute. But, I'm going to guess that this does indeed have to do with your load balancer. You're loading a `.js` file that has a MIME type of `text/html`, which should be sent with `application/javascript`.

Has anything changed with your load balancing setup?

Do you have a custom webpack config? I'll bump this over to a Discussion but we're happy to keep helping here. Just let me know.

**Reply by notchris** [reply]
I don't think we're managing the mime type, I think it's a 404 because of the unique string on the chunked files. I could be wrong though. We haven't changed anything in the load balancer from what I can see. It's just very odd as it was working correctly on Friday and no ones been in the office since.

**Reply by notchris** [reply]
Also, we do not have a custom webpack config

**Reply by notchris** [reply]
![Screenshot 2023-03-13 125616](https://user-images.githubusercontent.com/1944433/224771800-a526ec10-a38c-4022-bca7-7475f42c1f3e.png)
